keyboards/gboards/gergoplex/keymaps/default/rules.mk

```diff
@@ -3,7 +3,7 @@
 # Make sure you have dfu-programmer installed!
 #----------------------------------------------------------------------------
 # Firmware options
-MOUSEKEY_ENABLE        = no
```

Comment:
I feel so stupid 😄 
Enabled it [here](https://github.com/qmk/qmk_firmware/pull/13027/files#diff-b442860984712c178d99762a8e9e5e8d2a9692ca1ed37bf2ab0ed951e177a7f8R6), but this very line apparently overrides it.